scripts/update_python.py

```diff
@@ -153,6 +156,10 @@ def build_python():
     pybin = os.path.join(src_dir, 'install', 'usr', 'local', 'bin', 'python3')
     pip = os.path.join(src_dir, 'install', 'usr', 'local', 'bin', 'pip3')
     check_call([pybin, '-m', 'ensurepip', '--upgrade'])
+    # TODO: Potential bug: the following cmdline does not pin down a version
+    # of requests module, resulting in possibly different version of the module
+    # being installed on future runs. Switch to pip install requests==<version> to
+    # to download a pinned version.
```

Comment:
Maybe just add a version here instead of this comment?

Reply:
I don't know which version to add. Maybe if I get time to run this on a mac I'll check what version it was.